Frameworks/AEPSwiftUI/Sources/ContentCards/UIElement/AEPImage/AppStateManager.swift

```diff
@@ -0,0 +1,70 @@
+/*
+ Copyright 2024 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import AEPServices
+import Foundation
+import UIKit
+
+final class AppStateManager {
+    static let shared = AppStateManager()
+
+    /// The shared instance of `AppStateManager` used to access the singleton instance throughout the app.
+    private(set) var isAppInBackground = false
```

Comment:
because this is a property, i prefer naming it like a statement rather than a question.

`appIsInBackground` vs `isAppInBackground`

thoughts?

Reply:
I am thinking its better to keep `isAppInBackground` because of Swift's convention for Boolean properties